app/pyproject.toml

```diff
@@ -71,7 +71,7 @@ ingest-guru-cards = "src.ingest_guru_cards:main"
 ingest-policy-pdfs = "src.ingest_policy_pdfs:main"
 ingest-bem-pdfs = "src.ingest_bem_pdfs:main"
 ingest-edd-web = "src.ingest_edd_web:main"
-scrape-edd-web = "src.ingestion.scrape_edd_web:main"
```

Comment:
Did you intend to delete this line?

Reply:
...No I did not, I thought I was deleting my own line that I added earlier in development 🤦  thanks for catching!